docs/aserto-console/directory/create-permission.mdx

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_label: Creating Permissions
+title: Creating Permissions
+description: Creating permissions for relation types
+---
+
+# Permissions
+
+A relation type can optionally include named permissions. This provides a layer of indirection, which allows a relation (commonly referred to as a "role") to aggregate a set of permissions. 
+
+Note that the relationship between relations and permissions is "many to many".
+
+Aserto supports checking whether a user has a relationship to an object (e.g. an organization). It also supports checking whether a user has a permission on an object. This check returns "true" if ANY of the relations that the user has to this object contains the specified permission.
```

Comment:
would it be better to call these "relation types" instead of "relations" in these paras

Reply:
yes, agreed. reworked the text.